infrastructure/transform/network.tf

```diff
@@ -17,6 +17,85 @@ resource "azurerm_network_security_group" "databricks" {
   resource_group_name = var.core_rg_name
   location            = var.core_rg_location
   tags                = var.tags
+
+  # Add NSG rules manually to ensure they are in place before deploying a cluster instead of waiting for service delegation
```

Comment:
is there a way to de-duplicate this? (assuming the dbks NSG still gets deployed)